ext/CreateParametersExt.jl

```diff
@@ -96,8 +96,7 @@ function AutotrophicRespirationParameters(
         :live_stem_wood_coeff => :ηsl,
         :specific_leaf_density => :σl,
         :root_leaf_nitrogen_ratio => :μr,
-        :mol_CO2_to_kg_C_factor => :f1,
-        :relative_contribution_factor => :f2,
+        :relative_contribution_factor => :f,
```

Comment:
is the relative_contribution_factor the same as kg_C_to_mol_CO2_factor? Using `f` for both

Reply:
It might be clearer (even if long) to call this `kgC_to_molCO2`

Reply:
As discussed, this parameter is not a conversion but to say Rg is proportional to (An - Rpm). 
We will give it a different name to avoid confusion with the f in Heterotrophic params